From 9a2987d2af548d1d118d9f3ae0140c2cf4a03c6a Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Sat, 10 Oct 2020 14:35:24 +0200
Subject: [PATCH 34/40] ramips: make target work with trunk mac80211

The 19.07 tree is missing crypto support, which trunk mac80211 depends on.

Signed-off-by: John Crispin <john@phrozen.org>
---
 target/linux/ramips/mt76x8/config-4.14 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/linux/ramips/mt76x8/config-4.14 b/target/linux/ramips/mt76x8/config-4.14
index 00de576855..265105ef90 100644
--- a/target/linux/ramips/mt76x8/config-4.14
+++ b/target/linux/ramips/mt76x8/config-4.14
@@ -47,6 +47,7 @@ CONFIG_CPU_R4K_FPU=y
 CONFIG_CPU_SUPPORTS_32BIT_KERNEL=y
 CONFIG_CPU_SUPPORTS_HIGHMEM=y
 CONFIG_CPU_SUPPORTS_MSA=y
+CONFIG_CRYPTO_HASH=y
 CONFIG_CRYPTO_RNG2=y
 CONFIG_CRYPTO_WORKQUEUE=y
 CONFIG_CSRC_R4K=y
-- 
2.25.1

